src/app/components/main-menu/main-menu.tsx

```diff
@@ -32,26 +29,36 @@ export function MainMenu(props: MainMenuProps) {
             <ul className={styles['drop']}>
               {props.portfolio.map((portfolioItem) => (
                 <li key={portfolioItem.url}>
-                  <Link to={`/${portfolioItem.url}`}>
+                  <Link to={`/portfolio/${portfolioItem.url}`}>
                     {portfolioItem.title}
                   </Link>
                 </li>
               ))}
             </ul>
           </li>
-          <li className={styles['menu_list']}>About me</li>
-          <li className={styles['menu_list']}>Faq</li>
-          <li className={styles['menu_list']}>Contacts</li>
+          <Link to="/aboutme" className={styles['menu_list']}>
+            About me
+          </Link>
+          <Link to="/faq" className={styles['menu_list']}>
+            Faq
+          </Link>
+          <Link to="/contacts" className={styles['menu_list']}>
+            Contacts
+          </Link>
```

Comment:
To ensure consistency, it is recommended to enclose each `Link` item within a `li` tag when using `ul`.